specifyweb/frontend/js_src/lib/components/InitialContext/treeRanks.ts

```diff
@@ -83,7 +83,7 @@ export const treeRanksPromise = Promise.all([
                       definition: treeDefinition,
                       ranks: await fetchRelated(
                         serializeResource(treeDefinition),
-                        'treeDefItems',
+                        'treeEntries',
```

Comment:
This should remain `treeDefItems`. 
`treeEntries` is _all_ of the Tree Nodes (i.e., Geography, Taxon, Lithostrat, Storage, ...) which exist within the trees. 

We made the TreeDefitems dependent on the TreeDefinition. 
`fetchRelated` is to specifically fetch resources through independent relationships on resources. 
Since `treeDefItems` is dependent and now returning inline with the treeDefinition, we do not need to make another request to fetch the ranks. 
Instead, we should now be able to use something like `get`, `getDependentResource`, or even `rgetCollection`